WcaOnRails/db/migrate/20170316030443_add_big_json_to_competitions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddBigJsonToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :big_json, :text
```

Comment:
Oh, good questions!

1. This actually isn't a cache. It's better to think of this as a place to store arbitrary json data associated with a competition. If you're curious about the "big picture" here, I think the best written explanation is the "Motivation" section of [this google doc](https://docs.google.com/document/d/1jiGjmBoG_-5uZOF3RicMKZmZEI7rcPaQ6ax5z785vVs/edit#heading=h.c33cxotr1syu).
2. This is not something I've ever thought about before! I'll look into pulling this out into a CompetitionBigJson table sounds like a good plan.